week2/homework/src/add.js

```diff
@@ -0,0 +1,15 @@
+const fs = require('fs');
+
+function add(toDoList) {
+  if (toDoList === undefined) {
+    console.log('please enter a valid value after the command add');
+  }
+ else {
+    fs.appendFile('./todoList.txt', toDoList + '\n', error => {
+      console.error(error);
```

Comment:
Do we want to log even when there is no error i.e. the error is null?
![image](https://user-images.githubusercontent.com/3776580/61170251-be550180-a566-11e9-89c8-dae4b614eeb6.png)

Reply:
I modify it